[Bug]: Attach process by name index out of range

### Steps To Reproduce

```
docker run --privileged --rm -it cribl/scope:dev-x86_64
top &
top &
scope attach top
Found multiple processes matching that name...
ID	PID	USER	SCOPED	COMMAND
1 	16 	root	false 	top
2 	17 	root	false 	top
Select an ID from the list:
```
When choosing `3`
```
3
panic: runtime error: index out of range [2] with length 2

goroutine 1 [running]:
github.com/criblio/scope/run.choosePid({0xc000424460, 0x2, 0xc0000dfb60?})
	/home/michalbiesek/Github/appscope/cli/run/attach.go:119 +0x296
github.com/criblio/scope/run.(*Config).Attach(0x2a71d00, {0xc0003c3560, 0x1, 0x1})
	/home/michalbiesek/Github/appscope/cli/run/attach.go:47 +0x315
github.com/criblio/scope/cmd.glob..func1(0x2a44180?, {0xc0003c3560, 0x1, 0x1})
	/home/michalbiesek/Github/appscope/cli/cmd/attach.go:80 +0x4c5
github.com/spf13/cobra.(*Command).execute(0x2a44180, {0xc0003c3540, 0x1, 0x1})
	/home/michalbiesek/Github/appscope/cli/.gomod/github.com/spf13/cobra@v1.1.1/command.go:854 +0x663
github.com/spf13/cobra.(*Command).ExecuteC(0x2a43ee0)
	/home/michalbiesek/Github/appscope/cli/.gomod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x39c
github.com/spf13/cobra.(*Command).Execute(...)
	/home/michalbiesek/Github/appscope/cli/.gomod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/criblio/scope/cmd.Execute()
	/home/michalbiesek/Github/appscope/cli/cmd/root.go:23 +0x45
main.main()
	/home/michalbiesek/Github/appscope/cli/main.go:19 +0xb4
```

### Environment

```markdown
- AppScope: 1.1.0
- OS: Ubuntu 18.04
- Architecture:x86_64/aarch64
```


### Requested priority

Low

### Relevant log output

_No response_

--- a/cli/run/attach.go
+++ b/cli/run/attach.go
@@ -111,9 +111,9 @@ func choosePid(procs util.Processes) int {
 	fmt.Println("Select an ID from the list:")
 	var selection string
 	fmt.Scanln(&selection)
-	i, err := strconv.Atoi(selection)
+	i, err := strconv.ParseUint(selection, 10, 32)
 	i--
-	if err != nil || i < 0 || i > len(procs) {
+	if err != nil || i >= uint64(len(procs)) {
 		util.ErrAndExit("Invalid Selection")
 	}
 	return procs[i].Pid

